lib/seahorn/Bmc.cc

```diff
@@ -439,6 +198,35 @@ void unsat_core(ZSolver<EZ3> &solver, const ExprVector &f, bool simplify,
     out.push_back(bind::fname(bind::fname(c))->arg(0));
 }
 
+template <>
```

Comment:
why `template<>`?

Reply:
this is a specialization of template `Out` using llvm::raw_ostream

Reply:
add a comment that this is a specialization, just like the comment above